x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -7,16 +7,78 @@
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../ftr_provider_context';
 import { deleteMetadataStream } from '../../../security_solution_endpoint_api_int/apis/data_stream_helper';
-
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const pageObjects = getPageObjects(['common', 'endpoint', 'header', 'endpointPageUtils']);
   const esArchiver = getService('esArchiver');
   const testSubjects = getService('testSubjects');
 
+  const expectedData = [
+    [
+      'Hostname',
+      'Agent Status',
+      'Integration',
+      'Configuration Status',
+      'Operating System',
+      'IP Address',
+      'Version',
+      'Last Active',
+    ],
+    [
+      'cadmann-4.example.com',
+      'Error',
+      'Default',
+      'Failure',
+      'windows 10.0',
+      '10.192.213.130, 10.70.28.129',
+      '6.6.1',
+      'Jan 24, 2020 @ 16:06:09.541',
+    ],
+    [
+      'thurlow-9.example.com',
+      'Error',
+      'Default',
+      'Success',
+      'windows 10.0',
+      '10.46.229.234',
+      '6.0.0',
+      'Jan 24, 2020 @ 16:06:09.541',
+    ],
+    [
+      'rezzani-7.example.com',
+      'Error',
+      'Default',
+      'Failure',
+      'windows 10.0',
+      '10.101.149.26, 2606:a000:ffc0:39:11ef:37b9:3371:578c',
+      '6.8.0',
+      'Jan 24, 2020 @ 16:06:09.541',
+    ],
+  ];
+
   describe('endpoint list', function () {
     this.tags('ciGroup7');
     const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms));
 
+    describe('when initially navigating to page', () => {
+      before(async () => {
+        await pageObjects.endpoint.navigateToEndpointList();
+      });
+      after(async () => {
+        await deleteMetadataStream(getService);
+      });
+
+      it('finds no data in list and prompts onboarding to add policy', async () => {
+        await testSubjects.exists('emptyPolicyTable');
+      });
+
+      it('finds data after load and polling', async () => {
+        await esArchiver.load('endpoint/metadata/api_feature', { useCreate: true });
+        await sleep(10000);
```

Comment:
is there anything else we can rely on other than a sleep? Maybe wait for the count of items on the table to change?

Reply:
Is there a way to increase timeout for `await` on getting elements?  This `sleep` is the default poll time, so it's not entirely arbitrary.

Reply:
yes, I think most "waitfor" type of pageElement allow for a second argument of `options` that you can then control the retry timeout